CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.65.0 - 2022-05-16
+
+What's new:
+- API expansion to support Import from UI (#3233)
+- Airflow hook wait for DAG completion (#3231)
+- Friendlier post-setup page (#3318)
+- Show skipped hooks for failed action run (#3359)
+- Import from UI (#3352) 
+- Additional documentation for user management (#3320)
```

Comment:
Documentation changes should not be included in the changelog as they have different release cadence.

Reply:
:+1: 
Does the `include-changelog` a mistake, or should I be aware of that and exclude manually?

Reply:
It's a mistake